Misc/NEWS.d/next/Tests/2019-05-04-21-25-19.bpo-36782.h3oPIb.rst

```diff
@@ -0,0 +1,9 @@
+Created tests for these C API functions 
+- :c:func:`PyDate_FromDate`. 
+- :c:func:`PyDateTime_FromDateAndTime`. 
+- :c:func:`PyDateTime_FromDateAndTimeAndFold`. 
+- :c:func:`PyTime_FromTime`. 
+- :c:func:`PyTime_FromTimeAndFold`. 
+- :c:func:`PyDelta_FromDSU`. 
+
+Patch by Edison Abahurire.
```

Comment:
`Patch by Edison Abahurire` is fine on the same line. no need to add a new blank line.